charts/redpanda/values.yaml

```diff
@@ -252,6 +271,120 @@ storage:
     labels: {}
     # Additional annotations to apply to the created PersistentVolumeClaims.
     annotations: {}
+  # Tiered Storage helps to lower storage costs by offloading log segments to cloud storage.
+  # You can specify the amount of local storage you want to retain in local storage.
+  # You don't need to specify which log segments you want to move because Redpanda moves them automatically
+  # based on cluster-level configuration properties. Redpanda Tiered Storage indexes where data is offloaded,
+  # so it can retrieve the data when you need it.
+  #
+  # REF: https://docs.redpanda.com/docs/manage/tiered-storage/#tiered-storage-configuration-properties
+  #
+  # Topic configuration flags override any cluster-level configuration. For example, on new topics,
+  # if `cloud_storage_enable_remote_write` is set to `true`, you can set `redpanda.remote.write` to `false`
+  # to turn it off for a particular topic. Or, if you have Tiered Storage turned off for the cluster, you can enable it
+  # for individual topics with the topic flags. When cluster-level properties are changed,
+  # the changes apply only to new topics, not existing topics.
+  #
+  # REF: https://docs.redpanda.com/docs/manage/tiered-storage/#enable-tiered-storage-for-a-topic
+  tieredConfig:
+    # Global flag that enables Tiered Storage feature if license is provided.
+    # Default is false.
+    cloud_storage_enabled: false
+    # Cluster wide config option to enable data upload from Redpanda to cloud storage if set to true.
+    cloud_storage_enable_remote_write: true
+    # Cluster wide config option to enable Redpanda to fetch data from cloud storage if set to true.
+    cloud_storage_enable_remote_read: true
+    # This configuration option is available starting from 22.3.X
+    #
+    # Possible values:
+    # `config` - If IAM roles are not available, specify credentials in the cluster configuration file.
+    # `aws_instance_metadata` - For an AWS EC2 instance, use the instance metadata API from AWS.
+    # `sts` - For AWS on Kubernetes, use the Secure Token Service (STS).
+    # `gcp_instance_metadata` - For a VM running on GCP, or for Google Kubernetes Engine (GKE), use the instance metadata API from GCP.
+    #
+    # REF: https://docs.redpanda.com/docs/manage/security/iam-roles/#configuring-iam-roles
+    cloud_storage_credentials_source: config
+    # Cloud storage region. Required.
+    cloud_storage_region: ""
```

Comment:
Users may have the expectation that by enabling tiered storage and adding a bucket name in `values.yaml`, tiered storage will be enabled. But these changes also require the user to create the bucket, since we aren't creating the bucket via the helm chart.

Maybe we could add a comment (either close to this variable or at the start of the tiered storage config section) that mentioned the prerequisite step of creating a bucket.

Reply:
Sure I can add the prerequisite step, but this will mimic our doc. This contradict with other review and I'm not sure what your comment is about this thread https://github.com/redpanda-data/helm-charts/pull/278#discussion_r1072782702

Reply:
I just expect that users will assume they can enable tiered storage by making modifications in `values.yaml`, and if we don't let them know that's not all they need to do by some comment in this file, then there will be users who run into an error around tiered storage hopefully will reach out via some channel for help rather than just deciding they either 1) don't need tiered storage or 2) don't need Redpanda.

I'll make a comment in the thread you link to.

Reply:
This is interesting, but our docs https://docs.redpanda.com/docs/manage/tiered-storage doesn't mention that bucket needs to be created before any configuration changes.